models/users.js

```diff
@@ -143,6 +143,8 @@ const fetchPaginatedUsers = async (query) => {
     const size = parseInt(query.size) || 100;
     const doc = (query.next || query.prev) && (await userModel.doc(query.next || query.prev).get());
     let dbQuery = (query.prev ? userModel.limitToLast(size) : userModel.limit(size)).orderBy("username");
+    dbQuery = dbQuery.where("roles.archived", "==", false);
```

Comment:
still, the same issue that I mentioned first you will get 100 users then you will be filtering from them so it would return 100-n users but we expect 100 users not 100-n

Reply:
done